DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/svd/src/print_matrix.hpp

```diff
@@ -0,0 +1,70 @@
+#ifndef __PRINT_MATRIX_HPP__
+#define __PRINT_MATRIX_HPP__
+
+#include <iostream>
+#include <vector>
+
+namespace svd_testbench_tool {  // not for kernel code
+
+template <typename TT>
+void print_matrix(std::vector<std::vector<TT>> mat_A)
+{
+    // if its a floating point type number, fixed the print precision
```

Comment:
```suggestion
    // if its a floating point type number, fix the print precision
```

Reply:
This is going to make Coverity complain.
If possible avoid altering `cout` formatting entirely

Reply:
ok, i can use local string manipulation instead